sagemaker-jumpstart/object_detection/JumpStart_Object_Detection.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<p>Branding: ensure that all markdown uses are capitalized <code>JumpStart</code> - avoid "Jumpstart", "jumpstart", etc</p><p> </p><p>Grammar: <code>a Object Detection model</code> -&gt; <code>an Object Detection model</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done.</p>